addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -452,6 +461,160 @@ QUnit.test('do not post message with "Enter" keyboard shortcut', async function
     );
 });
 
+QUnit.test("Display suggested recipients list", async function (assert) {
+    assert.expect(3);
+
+    this.data['res.partner'].records.push({ id: 100, email: 'john@jane.be', display_name: 'John Jane' });
```

Comment:
Already commented that `email`/`display_name` should be double quotes, and data setup with multiple fields should be on multi-line for readability. This applies to whole diff of PR.